plumbing/hash/hash.go

```diff
@@ -4,12 +4,25 @@ package hash
 
 import (
 	"crypto"
+	"errors"
 	"fmt"
 	"hash"
 
+	format "github.com/go-git/go-git/v6/plumbing/format/config"
 	"github.com/pjbgf/sha1cd"
 )
 
+const (
+	SHA1Size      = 20
+	SHA1HexSize   = SHA1Size * 2
+	SHA256Size    = 32
+	SHA256HexSize = SHA256Size * 2
+)
```

Comment:
These are redefined here. We could use `format.SHA1Size` etc

Reply:
This was part of the sha256 conditional build, which is no longer needed. All this references have now been removed.